support-frontend/assets/pages/paper-subscription-checkout/helpers/paymentProviders.js

```diff
@@ -30,7 +90,17 @@ function onPaymentAuthorised(paymentAuthorisation: PaymentAuthorisation, dispatc
   };
 
   dispatch(setFormSubmitted(true));
-  Promise.resolve({ paymentStatus: 'success' }).then(handleSubscribeResult);
+
+  const data = buildRegularPaymentRequest(state, paymentAuthorisation);
+
+  postRegularPaymentRequest(
+    routes.digitalSubscriptionCreate,
```

Comment:
If this route is used by both products (and it's good if it is), then it should change to `subscriptionCreate`

Reply:
oh yes I will fix that